### PR TITLE
#166581814 Fix heroku review apps

### PR DESCRIPTION
**Pull request by Emmanuel-Dominic**
#### What does this PR do?
- [x] Fix Heroku functionality for review apps

#### Description of Task to be completed?
- [x] create the app.json file
- [x] setup the app.json configurations
- [x] ensure review apps are directed to the pull requests

#### What are the relevant pivotal tracker stories?
[#166581814](https://www.pivotaltracker.com/n/projects/2350327/stories/166581814)

### CODE REVIEW

**Review by reifred — APPROVED** [review]
Thanks @Emmanuel-Dominic, for fixing this. It was giving us a hard time to test our features without review apps.

File: requirements.txt [hunk 1]
```diff
@@ -18,11 +18,10 @@ django-sortedm2m==1.5.0
 django-bower==5.2.0
 invoke==1.2.0
 requests==2.21.0
-whitenoise==3.3.1
+whitenoise==4.1.2
```
Comment (Rhytah) [inline requirements.txt:21]: @Emmanuel-Dominic any particular reason for changing this version to a higher version?

Comment (Emmanuel-Dominic) [reply requirements.txt:21]: Feedback worked on, i have et it back to default the previous version.

**Review by Rhytah — APPROVED** [review]
@Emmanuel-Dominic thanks we really needed this change for the review apps to work without distorting deployment. As opposed to before when we had to always deploy branches

**Review by Manorlds-Eaglespark — APPROVED** [review]
Tests are running and the code is clean. Good work.

**Pull request merged**